### PR TITLE
Adjust the PUE server URL to the new domain

### PR DESCRIPTION
**Pull request by sc0ttkclark**

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
Waiting to confirm if this change will go into a release and what the ticket number will be.

File: src/Tribe/PUE/Checker.php [hunk 1]
```diff
@@ -235,7 +235,7 @@ private function set_slug( $slug = '' ) {
 		 * @return string
 		 */
 		public function get_pue_update_url() {
-			$pue_update_url = 'https://pue.tri.be';
+			$pue_update_url = 'https://pue.theeventscalendar.com';
```
Comment (tr1b0t) [inline src/Tribe/PUE/Checker.php:238]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 4c094b58f6ab557b0bbcd871f1c40006</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)

**Comment by zbtirrell** [comment]
let's do it!

**Pull request merged**